maven_lambda/__init__.py

```diff
@@ -145,3 +267,7 @@ def generate_checksums(data):
         'md5': hashlib.md5(data).hexdigest(),
         'sha1': hashlib.sha1(data).hexdigest(),
     }
+
+
+def is_snapshot(key):
+    return '-SNAPSHOT/' in key
```

Comment:
Not sure if the trailing backslash makes sense to be in the check. It doesn't hurt for sure, but thought it's cleaner in its absence.

Reply:
Okay. Done!